From 5e35f43236f8a1763e02ef8738f5fe8878959fb8 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 12 Nov 2014 15:40:56 +0100
Subject: [PATCH 003/158] avrdude: update to 6.1

change dependencies to libftdi1:
build complains about missing libftdi1 but not
if both libftdi are built but not if only the newer one is selected

add dependency on libelf1

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/avrdude/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/utils/avrdude/Makefile b/utils/avrdude/Makefile
index 015f0f4..cb717de 100644
--- a/utils/avrdude/Makefile
+++ b/utils/avrdude/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=avrdude
-PKG_VERSION:=6.0.1
+PKG_VERSION:=6.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/avrdude
-PKG_MD5SUM:=346ec2e46393a54ac152b95abf1d9850
+PKG_MD5SUM:=9db8c25b935d34234b9b1ba16ad55fd5
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -21,8 +21,8 @@ define Package/avrdude
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=AVR Downloader/UploaDEr
-  URL:=http://www.bsdhome.com/avrdude/
-  DEPENDS:=+libncurses +libreadline +libusb-compat +libftdi
+  URL:=http://www.nongnu.org/avrdude/
+  DEPENDS:=+libncurses +libreadline +libusb-compat +libftdi1 +libelf1
 endef
 
 define Package/avrdude/description
-- 
2.4.0

